server/docker-compose.makefile.yml

```diff
@@ -29,6 +29,7 @@ services:
     container_name: mattermost-minio
     ports:
       - "9000:9000"
```

Comment:
Do we still need the 9000 port?

Reply:
Well, that's the proper S3 API our app would use, so I'd say yes.

Reply:
Ah sorry, this is the web UI! Got confused :P